src/map.cpp

```diff
@@ -6999,6 +7001,173 @@ void map::rotten_item_spawn( const item &item, const tripoint &pnt )
     }
 }
 
+void map::actualize_snow( float new_snow_lvl )
```

Comment:
I suggest making `new_snow_lvl` here and in `map::actualize_snow_in_submap` below a constant reference, if it is not to be modified in the function body.

Reply:
Constant references make sense for larger structures, it's faster to pass simple types by value. But I can make `float new_snow_lvl` constant on the implementation for clarity.

Reply:
Yeah I realised and just edited my previous comment too. I think it is better to use `const float` to facilitate compiler optimisation.